galvan/src/main/java/org/terracotta/testing/master/GalvanStateInterlock.java

```diff
@@ -59,11 +60,25 @@ public GalvanStateInterlock(ContextualLogger logger, ITestWaiter sharedLockState
     this.sharedLockState = sharedLockState;
   }
 
+  public void ignoreServerCrashes() {
+    synchronized (this.sharedLockState) {
+      ignoreServerCrashes = true;
+      if (activeServer != null) {
+        activeServer.setCrashExpected();
+      }
+      passiveServers.forEach(process->process.setCrashExpected());
+      unknownRunningServers.forEach(process->process.setCrashExpected());
+      terminatedServers.forEach(process->process.setCrashExpected());
```

Comment:
Normally a server that is down from outside of galvan will result in a test failure.  This is giving us the ability to just let servers come and go by outside means and test failure will have to be determined another way.